Resources/Private/Templates/Authentication/ShowPasswordResetForm.html

```diff
@@ -3,20 +3,11 @@
 
   <f:flashMessages />
 
-<f:form.validationResults>
-  <f:if condition="{validationResults.flattenedErrors}">
-
-    <div class="alert alert-danger">
-      <p><f:translate key="resetPassword.errors">There where some errors with your input:</f:translate></p>
-      <ul class="errors">
-        <f:for each="{validationResults.flattenedErrors}" as="errors" key="propertyPath">
-          <li><f:translate key="{propertyPath}">{propertyPath}</f:translate>: {errors.0}</li>
-        </f:for>
-      </ul>
-    </div>
-
-  </f:if>
-</f:form.validationResults>
+  <f:form.validationResults>
```

Comment:
You know what? Let's simplify this even further. Let the templates only use `f:render partial` and put the whole `f:form.validationResults` viewhelper logic into that partial.